docs/list.md

```diff
@@ -355,6 +355,18 @@ Appwrite is a self-hosted backend-as-a-service platform that provides developers
 - **Issues**: [Eligible projects and issues](https://github.com/search?q=org%3Aturbot+topic%3Ahacktoberfest&type=repositories).
 - **Notes**: Read more details in the [announcement post](https://turbot.com/blog/2023/10/hacktoberfest-2023) and join our [Turbot community](https://turbot.com/community/join) `#contributors` channel with any questions.
 
+### W
+
+#### **WireMock**
+
+- **Swag**: T-Shirts, Stickers, Special swag.
+- **Requirements**:
+    - All projects in the official WireMock organization on GitHub.
```

Comment:
These are *where to contribute*, the requirements section needs the list of things one needs to do to be eligible to win the swag(s).

You could look into the other submissions at [hacktoberfestswaglist](https://hacktoberfestswaglist.com/list/) or check the [contributing guide](docs/contributing.md) to get help with this section.

Reply:
Hey @vimode, I removed that first line. Now is it okay?